labelCloud/view/gui.py

```diff
@@ -111,110 +111,110 @@ def __init__(self, control: "Controller") -> None:
 
         # MENU BAR
         # File
-        self.action_setpcdfolder = self.findChild(
+        self.action_setpcdfolder: QtWidgets.QAction = self.findChild(
             QtWidgets.QAction, "action_setpcdfolder"
-        )
-        self.action_setlabelfolder = self.findChild(
+        )  # type: ignore
```

Comment:
```
class QMainWindowWrapper(QtWidgets.QMainWindow):
    @overload
    def findChild(self, type: Type[QObjectT], name: str = ..., options: Union[QtCore.Qt.FindChildOptions, QtCore.Qt.FindChildOption] = ... ) -> QObjectT: 
        ...
    @overload
    def findChild(self, types: Tuple[Type[QtCore.QObject], ...], name: str = ..., options: Union[QtCore.Qt.FindChildOptions, QtCore.Qt.FindChildOption] = ...  ) -> QtCore.QObject: 
        ...
    def findChild(self, type, name, options):
        child = super().findChild(type, name = name, options = options)
        assert child is not None
        return child
```
I think the original typing of findChild is actually quite good. Mypy is just not sure if it can be None or not.
I think it could be solved by creating a wrapper and wrapping around the findChild and assert they are not None

Reply:
This could indeed by a solution.

I am thinking however if it is not cleaner to create Python files from the `*.ui`, which would solve the typing. Cause actually the `findChild()` is not really necessary as the attributes are already set according to the xml.

Having Python files could also increase startup time a bit. But currently there are some methods implemented like `eventFilter()` which would be overwritten every time the Python files are generated.

Reply:
Actually I just dropped the `findChild()` calls and am now using the variable names directly inside the xml and it works.

Thanks for pointing me there!

Reply:
ah that is great!